x-pack/plugins/security/public/management/api_keys/api_keys_grid/api_keys_grid_page.test.tsx

```diff
@@ -66,21 +66,17 @@ describe('APIKeysGridPage', () => {
   });
 
   const coreStart = coreMock.createStart();
-
-  const getViewProperties = () => {
+  const renderView = () => {
     const { docLinks, notifications, application } = coreStart;
-    return {
-      docLinks: new DocumentationLinksService(docLinks),
-      navigateToApp: application.navigateToApp,
-      notifications,
-      apiKeysAPIClient: apiClientMock,
-    };
+    return mountWithIntl(
+      <KibanaContextProvider services={{ application, docLinks }}>
```

Comment:
I would pass in the entire `coreStart` object into the kibana context provider here so that we don't get errors when a child component starts using e.g. notification or http service.

Reply:
I usually like to pass only what we use that makes it much easier to analyze affected code if we migrate/deprecate/remove anything (and we do this constantly). But in case of Core services I agree that's handy and tolerable, will change.